client/src/components/MapContainer.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import MapWithMarker from './MapWithMarker';
+
+class MapContainer extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			markers: [
+				{ lat: 31.705791, lng: 35.200657 },
```

Comment:
I thought you were planning to have this data come from a dummy API response JSON file?

Reply:
That is what we are planning to do in the next issue.